composable_kernel/include/utility/number.hpp

```diff
@@ -40,5 +40,67 @@ __host__ __device__ constexpr auto operator%(Number<X>, Number<Y>)
     static_assert(Y > 0, "wrong!");
     return Number<X % Y>{};
 }
+
+template <long_index_t N>
+using LongNumber = integral_constant<long_index_t, N>;
+
+template <typename Index0,
```

Comment:
```
template <typename Index0,
          Index0 X,
          typename Index1,
          Index1 Y>
__host__ __device__ constexpr auto operator+(integral_constant<Index0, X>,
                                             integral_constant<Index1, Y>)
{
    constexpr auto Z = X + Y;
    return integral_constant<decltype(Z), Z>{};
}
```

Reply:
That's indeed better. I moved the operators into integral_constant.hpp, leaving number.hpp almost empty. Should we also move using Number together?